node build error after updating to latest

```
(py-dev) C:\dev\code\jupyterlab (master) (jupyterlab@0.2.0)
λ pip install --no-deps -e .
Obtaining file:///C:/dev/code/jupyterlab
    Complete output from command python setup.py egg_info:
    setup.py entered
    running egg_info
    running jsdeps
    3.8.9
<SNIP>
    > jupyterlab-extension@0.1.0 build:extension C:\dev\code\jupyterlab\jupyterlab
    > webpack

    Generating config...
    Generating bundles...
    ModuleParseError: Module parse failed: C:\dev\code\jupyterlab\jupyterlab\build\jupyterlab-shim.js Bad character escape sequence (91:23)
    You may need an appropriate loader to handle this file type.
    SyntaxError: Bad character escape sequence (91:23)
        at Parser.pp$4.raise (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2221:15)
        at Parser.pp$7.readHexChar (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:3037:26)
        at Parser.pp$7.readCodePoint (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2913:19)
        at Parser.pp$7.readEscapedChar (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:3005:45)
        at Parser.pp$7.readString (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2935:23)
        at Parser.pp$7.getTokenFromCode (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2724:19)
        at Parser.pp$7.readToken (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2477:17)
        at Parser.pp$7.nextToken (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2468:15)
        at Parser.pp$7.next (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:2413:10)
        at Parser.pp.eat (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:536:12)
        at Parser.pp$3.parseSubscripts (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1731:25)
        at Parser.pp$3.parseExprSubscripts (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1718:17)
        at Parser.pp$3.parseMaybeUnary (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1692:19)
        at Parser.pp$3.parseExprOps (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1637:21)
        at Parser.pp$3.parseMaybeConditional (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1620:21)
        at Parser.pp$3.parseMaybeAssign (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1597:21)
        at Parser.pp$3.parseExpression (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:1573:21)
        at Parser.pp$1.parseStatement (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:727:47)
        at Parser.pp$1.parseTopLevel (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:638:25)
        at Parser.parse (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:516:17)
        at Object.parse (C:\dev\code\jupyterlab\jupyterlab\node_modules\acorn\dist\acorn.js:3098:39)
        at Parser.parse (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack\lib\Parser.js:902:15)
        at DependenciesBlock.<anonymous> (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack\lib\NormalModule.js:104:16)
        at DependenciesBlock.onModuleBuild (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack-core\lib\NormalModuleMixin.js:310:10)
        at nextLoader (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack-core\lib\NormalModuleMixin.js:275:25)
        at C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack-core\lib\NormalModuleMixin.js:259:5
        at Storage.finished (C:\dev\code\jupyterlab\jupyterlab\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:38:16)
        at C:\dev\code\jupyterlab\jupyterlab\node_modules\graceful-fs\graceful-fs.js:78:16
        at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:439:3)
        at DependenciesBlock.<anonymous> (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack\lib\NormalModule.js:113:20)
        at DependenciesBlock.onModuleBuild (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack-core\lib\NormalModuleMixin.js:310:10)
        at nextLoader (C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack-core\lib\NormalModuleMixin.js:275:25)
        at C:\dev\code\jupyterlab\jupyterlab\node_modules\webpack-core\lib\NormalModuleMixin.js:259:5
        at Storage.finished (C:\dev\code\jupyterlab\jupyterlab\node_modules\enhanced-resolve\lib\CachedInputFileSystem.js:38:16)
        at C:\dev\code\jupyterlab\jupyterlab\node_modules\graceful-fs\graceful-fs.js:78:16
        at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:439:3)

    npm ERR! Windows_NT 6.1.7601
    npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:extension"
    npm ERR! node v6.2.0
    npm ERR! npm  v3.8.9
    npm ERR! code ELIFECYCLE
    npm ERR! jupyterlab-extension@0.1.0 build:extension: `webpack`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the jupyterlab-extension@0.1.0 build:extension script 'webpack'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the jupyterlab-extension package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     webpack
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs jupyterlab-extension
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls jupyterlab-extension
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     C:\dev\code\jupyterlab\jupyterlab\npm-debug.log

    npm ERR! Windows_NT 6.1.7601
    npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
    npm ERR! node v6.2.0
    npm ERR! npm  v3.8.9
    npm ERR! code ELIFECYCLE
    npm ERR! jupyterlab-extension@0.1.0 build: `npm run update && npm run build:extension`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the jupyterlab-extension@0.1.0 build script 'npm run update && npm run build:extension'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the jupyterlab-extension package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     npm run update && npm run build:extension
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs jupyterlab-extension
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls jupyterlab-extension
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     C:\dev\code\jupyterlab\jupyterlab\npm-debug.log

    npm ERR! Windows_NT 6.1.7601
    npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:serverextension"
    npm ERR! node v6.2.0
    npm ERR! npm  v3.8.9
    npm ERR! code ELIFECYCLE
    npm ERR! jupyterlab@0.2.0 build:serverextension: `cd jupyterlab && npm run build && cd ..`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the jupyterlab@0.2.0 build:serverextension script 'cd jupyterlab && npm run build && cd ..'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the jupyterlab package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     cd jupyterlab && npm run build && cd ..
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs jupyterlab
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls jupyterlab
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     C:\dev\code\jupyterlab\npm-debug.log
    C:\Python\envs\py-dev\lib\distutils\dist.py:261: UserWarning: Unknown distribution option: 'License'
      warnings.warn(msg)
    rebuilding js and css failed
    missing files: ['C:\\dev\\code\\jupyterlab\\jupyterlab\\build\\jupyterlab.css', 'C:\\dev\\code\\jupyterlab\\jupyterlab\\build\\jupyterlab.bundle.js']
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\dev\code\jupyterlab\setup.py", line 173, in <module>
        setup(**setup_args)
      File "C:\Python\envs\py-dev\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "C:\Python\envs\py-dev\lib\distutils\dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "C:\Python\envs\py-dev\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "C:\dev\code\jupyterlab\setup.py", line 55, in run
        raise e
      File "C:\dev\code\jupyterlab\setup.py", line 48, in run
        self.distribution.run_command('jsdeps')
      File "C:\Python\envs\py-dev\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "C:\dev\code\jupyterlab\setup.py", line 116, in run
        run('npm run build:serverextension')
      File "C:\dev\code\jupyterlab\setup.py", line 29, in run
        check_call(cmd.split(), shell=shell, cwd=cwd, stdout=sys.stdout, stderr=sys.stderr)
      File "C:\Python\envs\py-dev\lib\subprocess.py", line 581, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['npm', 'run', 'build:serverextension']' returned non-zero exit status 1
```

I am also seeing these errors:

```
ModuleNotFoundError: Module not found: Error: Cannot resolve 'file' or 'directory' ../jade/jade in /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/vue
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/lib/Compilation.js:229:38
    at onDoneResolving (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/lib/NormalModuleFactory.js:29:20)
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/lib/NormalModuleFactory.js:85:20
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/async/lib/async.js:726:13
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/async/lib/async.js:52:16
    at done (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/async/lib/async.js:241:17)
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/async/lib/async.js:44:16
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/async/lib/async.js:723:17
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/async/lib/async.js:167:37
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:24:19
    at onResolved (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:38:18)
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:127:10
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:191:15
    at applyPluginsParallelBailResult.createInnerCallback.log (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/Resolver.js:110:4)
    at loggingCallbackWrapper (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/createInnerCallback.js:21:19)
    at /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/tapable/lib/Tapable.js:134:6
    at Tapable.<anonymous> (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/DirectoryDescriptionFilePlugin.js:24:12)
    at Storage.finished (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:38:16)
    at ReadFileContext.callback (/Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/webpack/node_modules/enhanced-resolve/node_modules/graceful-fs/graceful-fs.js:78:16)
    at FSReqWrap.readFileAfterOpen [as oncomplete] (fs.js:303:13)
resolve file
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade.webpack.js doesn't exist
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade doesn't exist
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade.web.js doesn't exist
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade.js doesn't exist
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade.json doesn't exist
resolve directory
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade doesn't exist (directory default file)
  /Users/bgranger/github/jupyter/jupyterlab/jupyterlab/node_modules/jupyterlab/node_modules/codemirror/mode/jade/jade/package.json doesn't exist (directory description file)
```

Replacing all `path.` with `path.posix.` seemed to get a lot further but finally failed with:
```
      |   +-- async-each@1.0.0
      |   +-- glob-parent@2.0.0
      |   +-- is-binary-path@1.0.1
      |   | `-- binary-extensions@1.5.0
      |   +-- is-glob@2.0.1
      |   | `-- is-extglob@1.0.0
      |   `-- readdirp@2.1.0
      |     `-- set-immediate-shim@1.0.1
      `-- webpack-core@0.6.8
        `-- source-map@0.4.4
          `-- amdefine@1.0.0

    npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
    npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.14

    > jupyterlab@0.2.0 build:serverextension C:\dev\code\jupyterlab
    > npm run build:css && cd jupyterlab && npm run build && cd ..


    > jupyterlab@0.2.0 build:css C:\dev\code\jupyterlab
    > concurrently "npm run build:css:base" "npm run build:css:theme"

    [1]
    [1] > jupyterlab@0.2.0 build:css:theme C:\dev\code\jupyterlab
    [1] > lessc src/default-theme/index.less src/default-theme/index.css
    [1]
    [0]
    [0] > jupyterlab@0.2.0 build:css:base C:\dev\code\jupyterlab
    [0] > lessc src/basestyle/index.less src/basestyle/index.css
    [0]
    [0] npm run build:css:base exited with code 0
    [1] npm run build:css:theme exited with code 0

    > jupyterlab-extension@0.1.0 build C:\dev\code\jupyterlab\jupyterlab
    > npm run update && npm run build:extension


    > jupyterlab-extension@0.1.0 update C:\dev\code\jupyterlab\jupyterlab
    > rimraf node_modules/jupyterlab && npm install


    > jupyterlab@0.2.0 prepublish C:\dev\code\jupyterlab
    > npm run build


    > jupyterlab@0.2.0 build C:\dev\code\jupyterlab
    > npm run build:css && npm run build:src


    > jupyterlab@0.2.0 build:css C:\dev\code\jupyterlab
    > concurrently "npm run build:css:base" "npm run build:css:theme"

    [0]
    [0] > jupyterlab@0.2.0 build:css:base C:\dev\code\jupyterlab
    [0] > lessc src/basestyle/index.less src/basestyle/index.css
    [0]
    [1]
    [1] > jupyterlab@0.2.0 build:css:theme C:\dev\code\jupyterlab
    [1] > lessc src/default-theme/index.less src/default-theme/index.css
    [1]
    [0] npm run build:css:base exited with code 0
    [1] npm run build:css:theme exited with code 0

    > jupyterlab@0.2.0 build:src C:\dev\code\jupyterlab
    > tsc --project src && node scripts/copyfiles.js


    > jupyterlab@0.2.0 postinstall C:\dev\code\jupyterlab\jupyterlab\node_modules\jupyterlab
    > node scripts/dedupe.js


    > jupyterlab-extension@0.1.0 postinstall C:\dev\code\jupyterlab\jupyterlab
    > node ../scripts/dedupe.js

    jupyterlab-extension@0.1.0 C:\dev\code\jupyterlab\jupyterlab
    `-- jupyterlab@0.2.0

    npm WARN optional Skipping failed optional dependency /chokidar/fsevents:
    npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.14

    > jupyterlab-extension@0.1.0 build:extension C:\dev\code\jupyterlab\jupyterlab
    > webpack

    Generating config...
    fs.js:951
      return binding.readdir(pathModule._makeLong(path), options.encoding);
                     ^

    Error: ENOENT: no such file or directory, scandir 'C:\dev\code\jupyterlab\jupyterlab\lib'
        at Error (native)
        at Object.fs.readdirSync (fs.js:951:18)
        at _walkSync (C:\dev\code\jupyterlab\jupyterlab\node_modules\walk-sync\index.js:64:18)
        at Function.entries (C:\dev\code\jupyterlab\jupyterlab\node_modules\walk-sync\index.js:40:10)
        at Object.createShim (C:\dev\code\jupyterlab\jupyterlab\node_modules\jupyterlab\scripts\extension_helpers.js:110:26)
        at Object.<anonymous> (C:\dev\code\jupyterlab\jupyterlab\webpack.config.js:30:22)
        at Module._compile (module.js:556:32)
        at Object.Module._extensions..js (module.js:565:10)
        at Module.load (module.js:473:32)
        at tryModuleLoad (module.js:432:12)

    npm ERR! Windows_NT 6.1.7601
    npm ERR! argv "C:\\bin\\nodejs\\node.exe" "C:\\bin\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:extension"
    npm ERR! node v6.4.0
    npm ERR! npm  v3.10.3
    npm ERR! code ELIFECYCLE
    npm ERR! jupyterlab-extension@0.1.0 build:extension: `webpack`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the jupyterlab-extension@0.1.0 build:extension script 'webpack'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the jupyterlab-extension package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     webpack
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs jupyterlab-extension
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls jupyterlab-extension
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     C:\dev\code\jupyterlab\jupyterlab\npm-debug.log

    npm ERR! Windows_NT 6.1.7601
    npm ERR! argv "C:\\bin\\nodejs\\node.exe" "C:\\bin\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
    npm ERR! node v6.4.0
    npm ERR! npm  v3.10.3
    npm ERR! code ELIFECYCLE
    npm ERR! jupyterlab-extension@0.1.0 build: `npm run update && npm run build:extension`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the jupyterlab-extension@0.1.0 build script 'npm run update && npm run build:extension'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the jupyterlab-extension package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     npm run update && npm run build:extension
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs jupyterlab-extension
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls jupyterlab-extension
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     C:\dev\code\jupyterlab\jupyterlab\npm-debug.log

    npm ERR! Windows_NT 6.1.7601
    npm ERR! argv "C:\\bin\\nodejs\\node.exe" "C:\\bin\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build:serverextension"
    npm ERR! node v6.4.0
    npm ERR! npm  v3.10.3
    npm ERR! code ELIFECYCLE
    npm ERR! jupyterlab@0.2.0 build:serverextension: `npm run build:css && cd jupyterlab && npm run build && cd ..`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the jupyterlab@0.2.0 build:serverextension script 'npm run build:css && cd jupyterlab && npm run build && cd ..'.
    npm ERR! Make sure you have the latest version of node.js and npm installed.
    npm ERR! If you do, this is most likely a problem with the jupyterlab package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     npm run build:css && cd jupyterlab && npm run build && cd ..
    npm ERR! You can get information on how to open an issue for this project with:
    npm ERR!     npm bugs jupyterlab
    npm ERR! Or if that isn't available, you can get their info via:
    npm ERR!     npm owner ls jupyterlab
    npm ERR! There is likely additional logging output above.

    npm ERR! Please include the following file with any support request:
    npm ERR!     C:\dev\code\jupyterlab\npm-debug.log
    C:\Python\lib\distutils\dist.py:261: UserWarning: Unknown distribution option: 'License'
      warnings.warn(msg)
    rebuilding js and css failed
    missing files: ['C:\\dev\\code\\jupyterlab\\jupyterlab\\build\\jupyterlab.css', 'C:\\dev\\code\\jupyterlab\\jupyterlab\\build\\jupyterlab.bundle.js']
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\dev\code\jupyterlab\setup.py", line 173, in <module>
        setup(**setup_args)
      File "C:\Python\lib\distutils\core.py", line 148, in setup
        dist.run_commands()
      File "C:\Python\lib\distutils\dist.py", line 955, in run_commands
        self.run_command(cmd)
      File "C:\Python\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "C:\dev\code\jupyterlab\setup.py", line 55, in run
        raise e
      File "C:\dev\code\jupyterlab\setup.py", line 48, in run
        self.distribution.run_command('jsdeps')
      File "C:\Python\lib\distutils\dist.py", line 974, in run_command
        cmd_obj.run()
      File "C:\dev\code\jupyterlab\setup.py", line 116, in run
        run('npm run build:serverextension')
      File "C:\dev\code\jupyterlab\setup.py", line 29, in run
        check_call(cmd.split(), shell=shell, cwd=cwd, stdout=sys.stdout, stderr=sys.stderr)
      File "C:\Python\lib\subprocess.py", line 584, in check_call
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '['npm', 'run', 'build:serverextension']' returned non-zero exit status 1

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in C:\dev\code\jupyterlab\
```

--- a/scripts/extension_helpers.js
+++ b/scripts/extension_helpers.js
@@ -104,7 +104,7 @@ function createShim(modName, sourceFolder) {
   // Find the path to the module.
   var modPath = require.resolve(modName + '/package.json');
   sourceFolder = sourceFolder || 'lib';
-  modPath = path.join(path.dirname(modPath), sourceFolder);
+  modPath = path.posix.join(path.dirname(modPath), sourceFolder);
 
   // Walk through the source tree.
   var entries = walkSync.entries(modPath, {
@@ -113,9 +113,9 @@ function createShim(modName, sourceFolder) {
   });
   for (var i = 0; i < entries.length; i++) {
     // Get the relative path to the entry.
-    var entryPath = path.join(sourceFolder, entries[i].relativePath);
+    var entryPath = path.posix.join(sourceFolder, entries[i].relativePath);
     // Add an entries for each file.
-    lines.push('shim["' + entryPath + '"] = require("' + path.join(modName, entryPath) + '");');
+    lines.push('shim["' + entryPath + '"] = require("' + path.posix.join(modName, entryPath) + '");');
   }
   lines.push('module.exports = shim;');
 

